src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -580,16 +580,23 @@ static void addERR_ParamsCantBeWithModifier(BindingDiagnosticBag diagnostics, Sy
             }
         }
 
-        private static void ReportParameterErrors(
-            Symbol owner,
+
+        public static void ReportParameterErrors(
+            Symbol? owner,
             BaseParameterSyntax parameterSyntax,
-            ParameterSymbol parameter,
+            int ordinal,
+            bool isParams,
+            TypeWithAnnotations paramTypeWithAnnotations,
+            TypeSymbol paramType,
+            RefKind paramRefKind,
+            DeclarationScope paramScope,
+            Symbol? paramContainingSymbol,
```

Comment:
>paramContainingSymbol [](http://example.com/codeflow?start=20&length=21)

Is this different from `owner`? If not, consider removing.

Reply:
The lambda code doesn't make use of either of these arguments. However, when `MakeParameters` calls this function, it passes through the `owner` parameter it got passed, plus the containing symbol attached to a particular parameter (`ParameterSymbol.ContainingSymbol`). I don't have the knowledge to say if those would always be the same

Reply:
Ok, I think it is the same as `owner` but it seems fine to pass in as a separate parameter. Perhaps, call it `containingSymbol` though. :)